subworkflows/local/eggnog.nf

```diff
@@ -18,6 +18,7 @@ workflow EGGNOG {
         File test = new File(eggnogDB)
 
         if (! directory.exists()){
+            directory.mkdir()
```

Comment:
I should probably run this, because I don't see how the subworkflow knows what to name the directory? Based on `eggnog_dbpath`? But isn't that an optional parameter?